src/Nancy/Request.cs

```diff
@@ -316,5 +318,13 @@ private void RewriteMethod()
 
             this.Method = providedOverride.Single().Item2;
         }
+
+        private string DebuggerDisplay
+        {
+            get
+            {
+                return String.Format("{0} {1}", this.Method, this.Url);

```

Comment:
Ideally I'd like it to resemble a HTTP request as closely as possible, so `HTTP/1.1 GET http://localhost/app/123` would do it for me :+1: